ocs_sample_library_preview/BaseClient.py

```diff
@@ -143,3 +143,9 @@ def request(self, method: str, url: str, params=None, data=None, headers=None, *
         if not headers:
             headers = self.sdsHeaders()
         return self.__session.request(method, url, params=params, data=data, headers=headers, **kwargs)
+
+    def getCommunityRequestHeaders(self, community_id: str):
```

Comment:
This should probably be ```communityHeaders``` to match ```sdsHeaders``` and placed below it instead of at the bottom.

Reply:
Agreed, good point

Reply:
Updated the name and moved it next to sdsHeaders, added documentation as well. 

Thank you!